Yona/Yona/en.lproj/Localizable.strings

```diff
@@ -154,7 +149,7 @@
 "enter-first-name-validation" = "You need to enter a name";
 "enter-last-name-validation"  = "You need to enter a name";
 "enter-nickname-validation" = "You need to enter a nick name";
-"enter-number-validation" = "Number should be 9 digits";
+"enter-number-validation" = "Number should be 9 digits.It must start with a + sign, with no spaces between the digits";
```

Comment:
We're removing the spaces, so that doesn't need to be mentioned in the error message.
Insert a space before "It must".

Reply:
Done.